src/contextmenu.js

```diff
@@ -150,8 +152,11 @@ export class ContextMenu {
         clampTimeout = timeout(() => clampSubMenu(this.core.$root, ev));
       },
       show: (options) => (props, actions) => {
-        let {menu, position} = options;
-        if (position instanceof Event) {
+        let {menu, position, toggle} = options;
+        if (toggle && isActive) {
+          return actions.hide();
+        }
+        else if (position instanceof Event) {
```

Comment:
Else statements must be placed on the same line as the curly brace: `} else ...`

Reply:
Huh.

Reply:
As in

```
        if (toggle && isActive) {
          return actions.hide();
        } else if (position instanceof Event) {
```